.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+max-line-length = 120
+extend-ignore = E203, W503, E266
```

Comment:
I think these ignores can be removed, they seem ok to me.

Reply:
removed